Xamarin.Forms.Platform.Android/AppCompat/FormsAppCompatActivity.cs

```diff
@@ -101,18 +101,18 @@ static void RegisterHandlers()
 			RegisterHandler(typeof(Picker), typeof(AppCompat.PickerRenderer), typeof(PickerRenderer));
 			RegisterHandler(typeof(CarouselPage), typeof(AppCompat.CarouselPageRenderer), typeof(CarouselPageRenderer));
 
-			if (Forms.Flags.Contains(Flags.FastRenderersExperimental))
+			if (Forms.Flags.Contains(Flags.UseLegacyRenderers))
+			{
+				RegisterHandler(typeof(Button), typeof(AppCompat.ButtonRenderer), typeof(ButtonRenderer));
+				RegisterHandler(typeof(Frame), typeof(AppCompat.FrameRenderer), typeof(FrameRenderer));
+			}
+			else
 			{
 				RegisterHandler(typeof(Button), typeof(FastRenderers.ButtonRenderer), typeof(ButtonRenderer));
 				RegisterHandler(typeof(Label), typeof(FastRenderers.LabelRenderer), typeof(LabelRenderer));
 				RegisterHandler(typeof(Image), typeof(FastRenderers.ImageRenderer), typeof(ImageRenderer));
```

Comment:
Huh. Wonder why this isn't symmetric.. Why don't we have to register `AppCompat.LabelRenderer` and `AppCompat`ImageRenderer` like we did for button and frame.

Reply:
Yeah, crazy, right? That's because there is no `AppCompat.LabelRenderer` and `AppCompat.ImageRenderer`. Those are still using the legacy legacy renderers (Pre-AppCompat), and those are registered in the forwarders via attribute.

This isn't confusing at all. 🤦‍♀️